Table: Delete all client references to groupToNestedTableV2 flag, functionally GAing the transformation (#127152)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/table/index.md

## Nested tables

{{< admonition type="note" >}}
The nested table overrides feature is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.

> To use these features, enable the `nestedFramesFieldOverrides` feature toggles in your Grafana configuration file or contact Support.

Reviewer: ```suggestion
To use this feature, enable the `nestedFramesFieldOverrides` feature toggle in your Grafana configuration file or contact Support.
```

{{< admonition type="note" >}}

> The new transformation editor for nested tables and the nested table overrides feature are currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.

Reviewer: Should the transformation editor part on line 213 be removed given the change on line 215?

Author: yes! thank you I totally missed this one 😓

{{< admonition type="note" >}}

> The new transformation editor for nested tables and the nested table overrides feature are currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.

Reviewer: ```suggestion
The nested table overrides feature is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.
```

> To use these features, enable the `nestedFramesFieldOverrides` feature toggles in your Grafana configuration file or contact Support.

Reviewer: ```suggestion
To use this feature, enable the `nestedFramesFieldOverrides` feature toggle in your Grafana configuration file or contact Support.
```
